src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -222,6 +224,34 @@ def test_minigraph_vlan_interfaces(self):
         output = self.run_script(argument)
         self.assertEqual(output.strip(), "[('Vlan1000', '192.168.0.1/27'), 'Vlan1000']")
 
+    def test_minigraph_ecmp_fg_nhg(self):
+        argument = '-m "' + self.ecmp_graph + '" -p "' + self.mlnx_port_config + '" -v \"FG_NHG.values()|list\"'
+        output = self.run_script(argument)
+        self.assertEqual(output.strip(), "[{'bucket_size': 120}, {'bucket_size': 120}]")
+
+    def test_minigraph_ecmp_members(self):
+        argument = '-m "' + self.ecmp_graph + '" -p "' + self.mlnx_port_config + '" -v "FG_NHG_MEMBER.keys()|list|sort"'
+        output = self.run_script(argument)
+        self.assertEqual(output.strip(), "['200.200.200.1', '200.200.200.10', '200.200.200.2', '200.200.200.3', '200.200.200.4', '200.200.200.5',"
+                                         " '200.200.200.6', '200.200.200.7', '200.200.200.8', '200.200.200.9', '200:200:200:200::1', '200:200:200:200::10',"
+                                         " '200:200:200:200::2', '200:200:200:200::3', '200:200:200:200::4', '200:200:200:200::5', '200:200:200:200::6',"
```

Comment:
In the future can we add checks for the values being populated in these entries(FG_NHG, link, and bank), having only the keys tested prevents 100% test coverage.

Reply:
Will do so